src/transaction/transaction.rs

```diff
@@ -48,8 +51,13 @@ impl Transaction {
     /// txn.commit().await.unwrap();
     /// # });
     /// ```
-    pub async fn get(&self, _key: impl Into<Key>) -> Result<Value> {
-        unimplemented!()
+    pub async fn get(&self, key: impl Into<Key>) -> Result<Value> {
+        let key = key.into();
+        if let Some(value) = self.get_from_mutations(&key) {
```

Comment:
The raw API returns `Result<Option<Value>>` and I think the transactional one should too. That means `get_value` will need to be changed to handle deleted values properly.